Add conditional render/disable examples

evilkermit opened the pull request:
## Summary of Changes
<!-- Summarize the changes made in this PR -->
Adds two examples for conditional rendering and disabling of elements.

## Checklist
<!-- Ensure all relevant checks are completed before requesting a review -->
- [x] The PR has a clear and concise title
- [x] Code is self-documented and follows [style guidelines](https://calvera.ornl.gov/docs/dev/project_management/style_guidelines/).
- [x] Automated tests are written and pass successfully.
- [x] Regression tests (e.g. manually triggered system tests, manual GUI/tool tests, ...) are performed to make sure the PR does not break anything (when applicable)
- [x] Readme file is present and up-to-date.

## Documentation Updates
<!-- Indicate whether any external documentation was updated -->

## Additional Notes
<!-- Provide any additional information that might be relevant -->

jarrah42 commented on examples/conditional_rendering/view.py:
```diff
@@ -0,0 +1,58 @@
+                        # We can also use v_show to conditionally render a field. Note that when using v_show, you won't
+                        # be able to use v_else_if or v_else afterwards. There is a deeper discussion of the differences
+                        # between v_if and v_show in the Vue.js documentation: https://vuejs.org/guide/essentials/conditional#v-if-vs-v-show.
+                        vuetify.VBtn("Toggle Comments Field", click=self.view_model.toggle_comments)
```
Why are you using a VBtn and not InputField?

jarrah42 commented on examples/data_selector/view.py:
```diff
@@ -0,0 +1,54 @@
+                        # circumstances.
+                        NeutronDataSelector(
+                            v_model="data.selected_files",
+                            base_paths=["/HFIR", "/SNS"],
```
I would specify something that you know will exist, such as $HOME and /tmp.

jarrah42 commented on README.md:
```diff
@@ -6,9 +6,18 @@ This repository contains examples of how to build application functionalities in
+4. [Conditionally disabling elements](examples/conditional_disabling)
+5. [Changing Pydantic rules based on user input](examples/dynamic_pydantic_rules)
+6. [Complex Pydantic rules](examples/complex_pydantic_rules)
+7. [Using datafiles from the Analysis Cluster](examples/data_selector)
```
can we move this to a separate ORNL Only section?, ditto ONCat.

evilkermit commented:
@jarrah42 I've updated the PR. Created an ORNL-specific subdirectory for examples and added an example for the plain DataSelector. Let me know if you want to see any other changes!

jarrah42 reviewed (approved):
Looks good, thanks.
